dec2024/README.md

```diff
@@ -0,0 +1,38 @@
+## December 2024
+
+https://docs.google.com/spreadsheets/d/1Vtxp5BLweDPDooQoNhgOCYjCIBPRYIcOuyArGJRqOkI/edit?gid=0#gid=0
+
+### Running script
+
+Runs on node v18
+
+#### Locally
+
+Ensure your planx-new docker container is running locally.
+
+Populate the table `temp_data_migrations_audit` with flows, for example:
+
+```psql
+INSERT INTO temp_data_migrations_audit (flow_id, team_id)
+SELECT id, team_id
+FROM flows 
+WHERE team_id IN (1,2,3); 
+```
+
+Then run the script, which will fetch & update a flow from the audit table which has not been `updated` yet.
+
+```sh
+cd dec2024
+HASURA_ENV=local HASURA_SECRET=secret node index.js
+```
+
+#### Scheduled via crontab
```

Comment:
This is a small TODO for me - I'll copy my local crontab here for future reference / memory jog